doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+.. _ug_bt_qualification:
+
+Bluetooth qualification
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Every product using Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
+This condition must be met to perform the following actions:
+
+* Demonstrate that your products satisfy the requirements of the Bluetooth license agreements and specifications.
+* Use Bluetooth trademark and logos.
+* Sell products that use Bluetooth LE technology.
+
+As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.
+
+
+Pre-qualified design
+********************
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+The following product types are used and qualified by Nordic Semiconductor:
+
+* End product
+* Controller Subsystem product
+* Host Subsystem product
+* Profile Subsystem product
+* Component product
+
+All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems and Components created by Nordic Semiconductor are qualified by them by default.
+Some Profile Subsystems (e.g., :ref:`ug_bt_mesh` versions) are qualified as well.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.
```

Comment:
Is this statement correct? To be verified by Miles.
If so, how exactly can customers request qualifying a Profile?

Reply:
There are two possibilities there: they could qualify it themselves, or ask if Nordic can do it. It is much more likely that they would have to do it themselves than waiting around for Nordic to think about it.  So I would suggest that we change that sentence a bit.  My suggestion below:
_"If you want to use a Profile Subsystem that is not currently listed by Nordic, you will need to qualify it yourself, typically by running the tests with the PTS test tool available from the Bluetooth SIG."_

Reply:
I see, corrected, thank you!